command/git.go

```diff
@@ -325,72 +365,21 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 	}
 
 	//Apply patches if necessary
-	if conf.Task.Requester != evergreen.PatchVersionRequester {
-		return nil
-	}
-
-	logger.Execution().Info("Fetching patch.")
```

Comment:
Moved fetching the patch (332-338) earlier (286-292) so I could use the data therein to checkout the PR on the merge.